### ISSUE TITLE
want to display Description of each web pages in the index.html and other reports

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/VisualTestingByImagemagickInKatalonStudio#31]
I made a small change to Test Cases/CURA/visitSite :
https://github.com/kazurayam/VisualTestingInKatalonStudio/commit/6a7fc031548b9e44ca01e11ce790cb0834b9702e

That test case script now puts the `MaterialDescription` to each Materials as screenshot. The description is something like "Home page", "Appointment Input Form", etc. Literal descriptions what the page is.

Now I want to see the Material Descriptions are displayed in the Materials/index.html

![MaterialDescriptionDisplayed](https://user-images.githubusercontent.com/4467417/88635109-3f3f5680-d0f2-11ea-93f3-92958791716e.png)

This enables me annotating anonymous description to each indivisual URL's screenshots. The description would be useful to make the index.html report easier for the readers to understand.

**Comment by kazurayam** [comment]
The `MaterialDescription` information is linked to the Actual Image. I have to make `MaterialDescription` information to the DiffImage object. 

I need to extend the Materials library to enable this.

**Comment by kazurayam** [comment]
I checked how much work will be involved to change the Materials/index.html to display MaterialDescription. I found it would require more work than I expected. The com.kazurayam.materials.view.IndexerBase program is not aware of the comparison-result-bundle.json file at all. And it seems that it is difficult to change IndexerBase. The IndexerBase scans the directory tree as is without knowing which file is what.

**Comment by kazurayam** [comment]
tag 1.23.0 implemented this.

In the `Test Cases/CURA/visiteSite` script, now write code that calls `MaterialRepository#resolve*Path*` method with argument of type `MaterialDescription`. You are supposed to write any description what that screenshot is about. For example, a screenshot of `http://demoaut.katalon.com` can have a description `ホームページ` in my native language. The code will look like this.

```
// create one more screenshot file with name in Japanese
Path png2 = mr.resolveMaterialPath(
					GlobalVariable[MGV.CURRENT_TESTCASE_ID.getName()],
					'screenshots',
					"ホーム.png",
					new MaterialDescription("1", "ホームページ"))
```

And the Test Suite Collection `Test Suite/CURA/Execute_chronos` and `Test Suite/CURA/Execute_twins` will produce `Materials/index.html` where all "diff images" will be annotated with the description which you added in the `Test Cases/CURA/visitSite` script.

In short, the page shows the whatever description of each URL's screenshots (in your native language). The page looks like this.  Much easier to understand, isn't it?

<img width="1440" alt="スクリーンショット 2020-07-31 15 15 33" src="https://user-images.githubusercontent.com/4467417/89111116-e6bedf00-d48c-11ea-9f07-d55c1737bb2c.png">
